Update migration-tool.adoc (#30)
mulesoft/docs-beta: modules/ROOT/pages/migration-tool.adoc

. Run MMA on a Mule 3 project.
. Import the Mule 4 application that MMA produced into a supported version of Studio 7.
. Review the Migration report that MMA produced.

> . Perform a manual migration on what MMA could not migrate.

Reviewer: ```suggestion
. Manually migrate what MMA cannot migrate automatically.
```

[[supported_migrations]]
== Supported Components

> *Note* : This list can be enhanced during time adding the support of new components

Reviewer: ```suggestion
*Note* : This list is subject to change and grow over time with the development of new MMA tasks for migrating additional connectors or components.
```

xref:migration-tool-procedure#datamapper[Migrating from the DataMapper to DataWeave].
* Custom types defined in Studio
* `<expression-component>`

> * Any other connectors or module not mentioned before.

Reviewer: ```suggestion
* Any unlisted connectors or modules.
```
